gitlab/resource_gitlab_project_hook.go

```diff
@@ -122,6 +122,11 @@ func resourceGitlabProjectHookRead(d *schema.ResourceData, meta interface{}) err
 
 	hook, _, err := client.Projects.GetProjectHook(project, hookId)
 	if err != nil {
+		if is404(err) {
```

Comment:
Same comment about resource not having an importer.

Reply:
same comment as well, let me know!